### PR TITLE
Add 'download csv' button to imports

### PR DESCRIPTION
**Pull request by guergana**
Bug: T304794

### CODE REVIEW

File: public/mix-manifest.json [hunk 1]
```diff
@@ -1,7 +1,10 @@
 {
     "/js/app.js": "/js/app.js",
     "/css/app.css": "/css/app.css",
+    "/images/download-stats-black.svg": "/images/download-stats-black.svg",
+    "/images/download-stats-white.svg": "/images/download-stats-white.svg",
     "/images/download-stats.svg": "/images/download-stats.svg",
```
Comment (micgro42) [inline public/mix-manifest.json:6]: question: As you renamed one usage of the `download-stats.svg` file to `download-stats-white.svg`, is this line here still needed?

File: public/mix-manifest.json [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
     "/images/mismatch-finder-logo.svg": "/images/mismatch-finder-logo.svg",
-    "/images/user.svg": "/images/user.svg"
+    "/images/user.svg": "/images/user.svg",
+    "/images/wikidata-logo.svg": "/images/wikidata-logo.svg"
```
Comment (micgro42) [inline public/mix-manifest.json:9]: suggestion: This line about the logo seems unrelated to this change.

I assume it was added by mistake and should probably be dropped?

File: resources/sass/app.scss [hunk 3]
```diff
@@ -250,6 +250,10 @@ dl.import-meta dd:not(:last-child) {
     padding-bottom: 16px;
 }
 
+dl.import-meta .download-csv {
+    float:right;
```
Comment (micgro42) [inline resources/sass/app.scss:254]: nitpick (non-blocking): missing space after the colon

(Though we really ought to have linting for that)

File: resources/views/importStatus.blade.php [hunk 4]
```diff
@@ -1,9 +1,10 @@
 <x-layout>
     <div role="group" class="button-group right">
-        <a class="button primary progressive" href="{{ route('store.imports-overview') }}"><img src="{{ asset('/images/download-stats.svg') }}" class="button-icon" />{{__('store-layout.button:download-stats')}}</a>
+        <a class="button primary progressive" href="{{ route('store.imports-overview') }}"><img src="{{ asset('/images/download-stats-white.svg') }}" class="button-icon button-white-fill" />{{__('store-layout.button:download-stats')}}</a>
```
Comment (micgro42) [inline resources/views/importStatus.blade.php:3]: Suggestion:  That extra class is probably not needed and can be dropped.

As I understand it, you can only affect SVGs with external CSS if they are literally in the DOM as HTML, not when they are used as an image in an `` tag.

Comment (guergana) [reply resources/views/importStatus.blade.php:3]: hi, yes, i mentioned that i am still working on this. ;)

File: resources/sass/app.scss (hunk 3 above)
Comment (guergana) [reply resources/sass/app.scss]: fixed!

File: public/mix-manifest.json (hunk 2 above)
Comment (guergana) [reply public/mix-manifest.json:7]: oh, yes, i don't know where this is coming from, since this file is automatically generated.

Comment (guergana) [reply public/mix-manifest.json]: removed!

**Comment by micgro42** [comment]
Looks good to me purely from inspecting the code.
As this does not have browser tests, we should also make sure we try out the functionality manually. I currently don't have a working setup locally, but maybe @chukarave could give it a quick spin, given that you also have done the work that created these routes and are familiar with all the edge-cases that might exist there :)

File: resources/lang/en/store-layout.php [hunk 5]
```diff
@@ -17,6 +17,7 @@
     'button:delete' => 'Delete',
     'button:confirm' => 'Confirm',
     'button:create' => 'Create',
+    'button:download-import-csv' => 'Download csv',
```
Comment (chukarave) [inline resources/lang/en/store-layout.php:20]: I know this is how it was written in Figma, but as it is an acronym i think we want to write it in upper case.

```suggestion
    'button:download-import-csv' => 'Download CSV',
```

**Comment by chukarave** [comment]
It looks good and works well! 
I wonder if we should a dd a media query to remove the float on mobile, as the button width squashes the `Import Status` label (This is Firefox, it looks better on Chrome but i still think it might make sense). Tagging @sai-san to have a look.
(with float)
image

(without)
image

File: resources/lang/en/store-layout.php (hunk 5 above)
Comment (guergana) [reply resources/lang/en/store-layout.php]: Oh, i just saw the suggestion, I have pushed the change already. Thanks!!

**Comment by sai-san** [comment]
> I wonder if we should add a media query to remove the float on mobile, as the button width squashes the `Import Status` label (This is Firefox, it looks better on Chrome but i still think it might make sense).
> (without) image

Apologies for the lack of responsive specifications here. I agree with  @chukarave's suggestion and think that the button should simultaneously become full-width and be placed on top of the Import Status card at the 720px media query, @width-breakpoint-tablet (the same that's altering the size and placement of the 'Download statistics button'). 
When placed at the top, the 'Download CSV' buttons should display a `margin-block-end` of 1em that separates them from the first <dt> ('Import status').

**Review by chukarave — APPROVED** [review]
Looks great, thank you!
